.github/workflows/fms-base.yml

```diff
@@ -0,0 +1,68 @@
+name: build-fms-base
+
+on:
+  workflow_dispatch:
+    inputs:
+      commit:
+        description: 'Commit in the FMS repo to build from.'
+        required: true
+        type: string
+        default: 'master'
+      version:
+        description: 'Debian package version to use for the package.'
+        required: true
+        type: string
+        default: 'v1.0'
+jobs:
+  build:
+    strategy:
+      max-parallel: 1
+      matrix:
+        arch: ['amd64']
+        debian_release: ['bullseye', 'bookworm']
+
+    runs-on: ubuntu-20.04
```

Comment:
Why not 24.04?

Reply:
Ah good spot, think this was an artefact of debugging a docker multi-platform issue. Will revert since I've dropped the arm64 builds for now.

Reply:
Ah nope, looks like it was this:
> The version '3.9.1' with architecture 'x64' was not found for Ubuntu 24.04.

Reply:
Same with 22.04 so will keep 22.02 and add a comment.

Reply:
OK, so I think I fixed on 3.9.1 to match Bullseye, which at the time was the primary distro I wanted my local pyenv to match. I'm not sure we necessarily need to be this restrictive going forward, but perhaps this PR isn't the place to fix that - I'll open a separate ticket to cover this and let's not make it a blocked for this.